Gemfile

```diff
@@ -6,6 +6,9 @@ if !ENV['CIRCLECI']
     gem "danger"
     gem "danger-xcode_summary"
     gem "cocoapods"
+    # fork by 'https://github.com/flix-tech/danger-code_style_validation.git'
+    # update the message level from fail to warn
+    gem 'danger-code_style_validation', :git => 'https://github.com/wrmswindmill/danger-code_style_validation.git'
```

Comment:
Why adding this?

Reply:
because of code-format-validator plugin too

Reply:
I didn't find where you invoke `code-format-validator`

Reply:
OK